ansys/mapdl/core/launcher.py

```diff
@@ -145,9 +145,28 @@ def port_in_use(port, host=LOCALHOST):
         except:
             return True
 
+def delete_ip_file(path):
+    """Delete the created 'mylocal.ip'."""
+    # path=''
+    file_name = os.path.join(path,'mylocal.ip') 
+    if os.path.exists(file_name):
+        # os.remove(file_name)
+        pass 
```

Comment:
Remove this

Reply:
Actually, I see why you have this in here (remove IP file when complete).  However, it doesn't seem to be actually removing the file since you've commented it out.

I'd to a try except to remove the file if it exists.

Reply:
It seems that the python code goes faster than the MAPDL code, hence Python does delete the file before the MAPDL process reaches it. One option will be to add more delay after the process is launch but I rather not do that. I prefer a text file around that slow down the whole process.

Reply:
does it makes sense?

Reply:
Yes, that makes sense.  In light of this, don't cleanup the file at the end.  Also, since we're using write mode, it doesn't matter if the file already exists.